Libs/Reflex/src/main/java/reflex/value/ReflexDateValue.java

```diff
@@ -70,7 +70,7 @@ public ReflexDateValue(LocalDate other, String calendar) {
         setupCalendarHandler();
     }
 
-    private static final DateTimeFormatter FORMATTER = DateTimeFormat.forPattern("yyyyMMdd").withZoneUTC();
+    private DateTimeFormatter FORMATTER = DateTimeFormat.forPattern("yyyyMMdd").withZoneUTC();
```

Comment:
I'm concerned about the use of UTC in this class. It means that the date changes in the middle of the afternoon. We should be able to specify the timezone, which means the formatter shouldn't be a static final. I'll revert it and log a ticket.